addons/vanilla-postgresql/templates/cmpv.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps.kubeblocks.io/v1
+kind: ComponentVersion
+metadata:
+  name: vanilla-postgresql
+  labels:
+    {{- include "vanilla-postgresql.labels" . | nindent 4 }}
+spec:
+  compatibilityRules:
+    - compDefs:
+        - {{ include "vanilla-postgresql-12.cmpdRegexpPattern" . }}
+      releases:
+        - 12.15.0
+    - compDefs:
+        - {{ include "vanilla-postgresql-14.cmpdRegexpPattern" . }}
+      releases:
+        - 14.7.0
+    - compDefs:
+        - {{ include "vanilla-postgresql-15.cmpdRegexpPattern" . }}
+      releases:
+        - 15.7.0
+    - compDefs:
+        - {{ include "vanilla-postgresql-supabase15.cmpdRegexpPattern" . }}
+      releases:
+        - supabase-15.6.1.138
+  releases:
+    - name: 12.15.0
+      serviceVersion: 12.15.0
+      images:
+        postgresql: {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.repository }}:{{ .Values.image.tags.major12.minor15 }}
+    - name: 14.7.0
+      serviceVersion: 14.7.0
+      images:
+        postgresql: {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.repository }}:{{ .Values.image.tags.major14.minor07 }}
+    - name: 15.7.0
+      serviceVersion: 15.7.0
+      images:
+        postgresql: {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.repository }}:{{ .Values.image.tags.major15.minor07 }}
+    - name: supabase-15.6.1.138
+      serviceVersion: 15.6.1138
+      images:
+        postgresql: {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.supabase.repository }}:{{ .Values.image.supabase.tags.major15.minor61138 }}
```

Comment:
the image tag in cmpv should be static, check them all

Reply:
done